server/example.env

```diff
@@ -1,24 +1,35 @@
 # Port number to run the server on
-export PORT=3030
+PORT=3030
 
 # Twilio account sid
-export ACCOUNT_SID=ACxxxxxxxxxxxxxx
+ACCOUNT_SID=ACxxxxxxxxxxxxxx
 
 # Twilio auth token
-export AUTH_TOKEN=xxxxxxxxxxxxxx
+AUTH_TOKEN=xxxxxxxxxxxxxx
 
 # Twilio API key
-export API_KEY_SID=SKxxxxxxxxxxxxxx
+API_KEY_SID=SKxxxxxxxxxxxxxx
 
 # Twilio API secret
-export API_KEY_SECRET=xxxxxxxxxxxxxx
+API_KEY_SECRET=xxxxxxxxxxxxxx
 
 # Twilio TwiML App sid
-export OUTGOING_APPLICATION_SID=APxxxxxxxxxxxxxx
+OUTGOING_APPLICATION_SID=APxxxxxxxxxxxxxx
 
 # Caller ID
-export CALLER_ID=+11234567890
+CALLER_ID=+11234567890
 
 # Push credential for incoming calls
 # Please see https://github.com/twilio/twilio-voice-react-native#incoming-calls
-export PUSH_CREDENTIAL_SID=xxxxxxxxxxxxxx
+PUSH_CREDENTIAL_SID=xxxxxxxxxxxxxx
+
+# Twilio TwiML Request Url
+TWIML_REQUEST_URL=http://xxxxxxxxxxxx.ngrok.io/twiml
```

Comment:
TWIML_REQUEST_URL is usually the customer's domain. This is also called voice request url in your twiml app and should be served via https. Let's update this section to something like the following.
```sh
# Twilio Voice Request URL configured in your TwiML App
VOICE_REQUEST_URL=https://mycompany.com/twiml
```

However, if you can validate the request using what Michael suggested [here](https://github.com/twilio/twilio-voice-react-native-app/pull/33/files#r1139342402), then no need for this change.

Reply:
Yes decided to go with Michael's suggestion, since then we no longer need the TwiML url